_redirects

```diff
@@ -1,3 +1,14 @@
+# The exclamation mark at the end of the HTTP code is to make sure
```

Comment:
Great with these comments! Thanks @giovannipizzi

Reply:
You're welcome! It took some time to figure it out, so I preferred to explain ;-)